spec/models/group_spec.rb

```diff
@@ -491,24 +491,23 @@
     Group.delete_all
     Group.refresh_automatic_groups!
 
-    groups = Group.includes(:users).to_a
-    expect(groups.count).to eq Group::AUTO_GROUPS.count
+    expect(Group.count).to eq Group::AUTO_GROUPS.count
 
-    g = groups.find { |grp| grp.id == Group::AUTO_GROUPS[:admins] }
-    expect(g.users.count).to eq g.user_count
-    expect(g.users.pluck(:id)).to contain_exactly(admin.id)
+    g = Group[:admins]
+    expect(g.human_users.count).to eq(g.user_count)
```

Comment:
This particular spec embodies the change. The user count should match the number of human users.